domain.yml

```diff
@@ -228,8 +232,6 @@ templates:
     - text: "You can’t speak to a human directly in this chat, but you can email us at hi@rasa.com."
   utter_no_speak:
     - text: "I'm afraid we can't chat then 😢"
-  utter_ask_whatspossible:
-    - text: "You can book a sales call or subscribe to our newsletter."
```

Comment:
i think this needs to stay because the chitchat intent looks for `utter_` + intent